packer_test/plugin_tests/init_test.go

```diff
@@ -1,6 +1,8 @@
-package packer_test
+package plugin_tests
```

Comment:
I see now why you chose core_test above. This feels like unneeded package nesting to split the tests. These are all Packer command tests. In fact it appears that this package offers nothing new besides renaming PackerTestSuite to PackerPluginTestSuite. Please let me know if I am missing something? 

That said, the tests for plugins should be renamed to better indicate what is being tested. For example `TestPackerInitForce` could be renamed to `TestPackerInitWithForceFlag`, which read on the screen would be 

`TestPackerInitWithForceFlag/installs_any_missing_plugins` or `TestPackerInitWithForceFlag/reinstalls_plugins_matching_version_constraints`